### PR TITLE
Add Clubhouse

### PR DESCRIPTION
**Pull request by fletcherdodd**

### CODE REVIEW

File: entries/c/clubhouse.com.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "Clubhouse": {
+    "domain": "clubhouse.com",
+    "contact": {
+      "twitter": "Clubhouse"
```
Comment (hkamran80) [inline entries/c/clubhouse.com.json]:
```suggestion
      "twitter": "Clubhouse",
      "form": "https://5nhc6u4b3x3.typeform.com/to/Asez2lqx"
```

I found this feature request form on [their support site](https://support.clubhouse.com/hc/articles/4405242707603).

**Review by hkamran80 — CHANGES_REQUESTED** [review]

File: entries/c/clubhouse.com.json (hunk 1 above)
Comment (fletcherdodd) [reply entries/c/clubhouse.com.json]: Good find! I could only find that 'submit a request' and all of those topics didn't really match suggesting/requesting 2FA.

Comment (hkamran80) [reply entries/c/clubhouse.com.json]: Happy to help!

**Review by hkamran80 — APPROVED** [review]

**Comment by hkamran80** [comment]
Thank you! Your contribution will be visible in around 10 minutes, although it may take longer due to [server and browser caching](https://github.com/2factorauth/twofactorauth/wiki/FAQ-Caching).

**Pull request merged**